/lc_serial_driver:
  ros__parameters:
    device_name: /dev/ttyACM0
    baud_rate: 115200
    flow_control: none
    parity: none
    stop_bits: "1"
    
    is_pitch_gain: true
    is_track: true 
    shoot_delay: 0.4
    shoot_delay_spin_: 0.2
    shoot_speed: 14.4

    x_gain: 0.0
    y_gain: 0.0
    pitch_gain_: 0.15
    
    
    # 龙格库塔法求解落点的迭代次数
    R_K_iter: 50
    # 使用迭代法求解pitch补偿的最大迭代次数
    max_iter: 10
    # 停止迭代的最小误差(单位m)
    stop_error: 0.001